src/renderers/shaders/ShaderChunk/lights_lambert_vertex.glsl

```diff
@@ -20,10 +20,10 @@ vLightFront = vec3( 0.0 );
 
 	for ( int i = 0; i < NUM_POINT_LIGHTS; i ++ ) {
 
-		IncidentLight directLight = getPointDirectLight( pointLights[ i ], geometry );
+		IncidentLight directIrradiance = getPointDirectIrradiance( pointLights[ i ], geometry );
 
-		float dotNL = dot( geometry.normal, directLight.direction );
-		vec3 directLightColor_Diffuse = PI * directLight.color;
+		float dotNL = dot( geometry.normal, directIrradiance.direction );
```

Comment:
`directIrradiance.direction` and `directIrradiance.color` do not make sense to me. Irradiance does not have a direction and color property. Lights do.

Perhaps 

    IncidentLight incidentLight = getPointLight(...)

    incidentLight.direction
    incidentLight.color
    incidentLight.irradiance